com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -78,6 +78,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed Intensity Multiplier not affecting realtime global illumination.
 - Fixed an exception when opening the color picker in the material UI (case 1307143).
 - Fixed lights shadow frustum near and far planes.
+- Fixed that the HDRP Wizard is only open when a SRP in use is of type HDRenderPipeline.
```

Comment:
I would argue that we can only say "The HDRP Wizard is only opened when a SRP in use is of type HDRenderPipeline."

Reply:
(we are in the category Fixed)